replication_chexnet_cuda_local.py

```diff
@@ -238,8 +236,6 @@ def eval_model(model, test_loader, logfile):
     y_test = y_test.cuda()
     y_pred = torch.FloatTensor()
     y_pred = y_pred.cuda()
-    criterion = nn.BCELoss()
```

Comment:
removed dup

Reply:
criterion = nn.BCELoss() here is not duplicated, since it is within eval_model, we need this to calculate the validation loss

Reply:
We already have it declared at line235

Reply:
agreed, this should be removed as line 235 already defined the criterion.